contracts/src/periphery/cert/libraries/EigenDACertVerificationLib.sol

```diff
@@ -58,6 +53,50 @@ library EigenDACertVerificationV2Lib {
 
     }
 
+    function decodeCert(bytes calldata data) internal pure returns (CT.EigenDACertV3 memory cert) {
+        return abi.decode(data, (CT.EigenDACertV3));
```

Comment:
do we not want to catch reverts here and return a StatusCode `INVALID_ABI_ENCODED_CERT` or something?

Reply:
Only if you want to deal with this nightmare https://forum.soliditylang.org/t/call-for-feedback-the-future-of-try-catch-in-solidity/1497

Personally, I think it is better to just accept reverts rather than try to work around this for initial versions.